build-system/compile/helpers.js

```diff
@@ -16,7 +16,7 @@ function getSourceRoot(options) {
       .replace(/\{version\}/g, internalRuntimeVersion)
       .replace(/([^/])$/, '$1/');
   }
-  if (options.fortesting) {
+  if (options.fortesting || !argv._.includes('dist')) {
     return 'http://localhost:8000/';
   }
```

Comment:
FMI: does removing the sourceMapRoot work as well?

Reply:
it does not, when I tried and put a `debugger` in amp.js it tries to find the map @ `http://localhost:8000/dist/src/amp.js` but I'm not sure why. I still see `# sourceMappingURL=amp.js.map` at the bottom of that file.